woocommerce/woocommerce
# Tax reports not showing correct totals

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #42682



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create two standard rates from `/wp-admin/admin.php?page=wc-settings&tab=tax&section=standard`
2. Create two orders and assign each standard rate you create in point 1 to the orders: rate 1 to the order 1 and rate 2 to the order 2.
3. Save both orders as `Completed`
4. In the `Analytics > Taxes` report the totals should be correctly calculated based on the items shown in the table <img width="777" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13334210/45a3fc8a-f728-4120-8bb3-50faea81f3ba">
5. After deleting one standard rate created in point 1, it should still be shown in the table and the totals should not be affected while the table should looks like in point 4



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
